src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java

```diff
@@ -116,7 +117,21 @@ private NotificationsResponse mapToNotificationsResponse(List<Notification> list
         List<NotificationInfo> notifications = list.stream()
             .map(notification -> toNotificationResponse(notification))
             .collect(toList());
-        return new NotificationsResponse(notifications.size(), notifications);
+
+        var sentNotificationsCount = notifications.stream()
+            .filter(n -> n.status.equals("SENT"))
+            .collect(Collectors.toList()).size();
+
+        var pendingNotificationsCount = notifications.stream()
+            .filter(n -> n.status.equals("PENDING"))
+            .collect(Collectors.toList()).size();
```

Comment:
in one loop ? like you did last time.

Reply:
and this code can  move to 'NotificationsResponse'
and  "SENT" and "PENDING" can be constants

Reply:
use PENDING.name(), SENT.name()

Reply:
And previously counts were derived in the constructor of the response class